src/main/include/utils/CommandUtils.h

```diff
@@ -323,6 +325,9 @@ static frc2::CommandPtr Intake2(IntakeSubsystem* intakeSubsystem,
 
   return (IntakeInInitial(intakeSubsystem)
               .ToPtr()
+              .AlongWith(frc2::InstantCommand([scoring] {
```

Comment:
No timeout? Could this cause a lockup?

Reply:
Will change after first match, good catch